clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/AllowLaunchAtomicOperation.groovy

```diff
@@ -61,28 +62,13 @@ class AllowLaunchAtomicOperation implements AtomicOperation<ResolvedAmiResult> {
     def sourceAmazonEC2 = amazonClientProvider.getAmazonEC2(description.credentials, description.region, true)
     def targetAmazonEC2 = amazonClientProvider.getAmazonEC2(targetCredentials, description.region, true)
 
+    def (ResolvedAmiResult resolvedAmi, ResolvedAmiLocation amiLocation) = new RetrySupport().retry({ ->
+      resolveAmi(targetCredentials, sourceAmazonEC2, targetAmazonEC2)
+    }, 5, 1000, false)
```

Comment:
these numbers OK? I copied from elsewhere.

Reply:
should be ok - 5 retries 1sec apart. I don't think we want to retry too much longer than that in here, if it turns out this isn't enough maybe the next step is to look at moving things up to orca.